modules/persistent-storage-csi-smb-cifs-limits.adoc

```diff
@@ -14,4 +14,6 @@ When Federal Information Processing Standards (FIPS) mode is enabled, the use of
 
 * Using HTTP proxy configuration to connect to outside of the cluster SMB servers is not supported by the CSI driver.
 +
-Since CIFS/SMB is a LAN protocol, and though it can be routed to subnets, it is not designed to be extended over the WAN, and does not support HTTP proxy settings. 
\ No newline at end of file
+Since CIFS/SMB is a LAN protocol, and though it can be routed to subnets, it is not designed to be extended over the WAN, and does not support HTTP proxy settings. 
+
```

Comment:
Add the fact that we don't support kerberos
"Kerberos based authentication is not supported."

Reply:
Fixed.